src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private final double firstLeg;
+    private final double secondLeg;
+    private final Color color;
```

Comment:
Consider using a more descriptive name for the 'color' field, such as 'figureColor', to improve readability and avoid potential confusion with local variables that might also be named 'color'.